src/components/shared-plot-options/CellColorEncodingOption.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Select from '@material-ui/core/Select';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { capitalize } from '../../utils';
+import { useStyles } from './styles';
+
+export default function CellColorEncodingOption(props) {
+  const {
+    observationsLabel,
+    cellColorEncoding,
+    setCellColorEncoding,
+  } = props;
+
+  const classes = useStyles();
+
+  const observationsLabelNice = capitalize(observationsLabel);
+
+  function handleColorEncodingChange(event) {
+    setCellColorEncoding(event.target.value);
+  }
+
+  return (
+    <TableRow>
+      <TableCell className={classes.labelCell} htmlFor="cell-color-encoding-select">
+        {observationsLabelNice} Color Encoding
+      </TableCell>
+      <TableCell className={classes.inputCell}>
+        <Select
+          native
+          value={cellColorEncoding}
+          onChange={handleColorEncodingChange}
+          inputProps={{
+            id: 'cell-color-encoding-select',
+          }}
```

Comment:
```suggestion
          inputProps={{
            style: { fontSize: '.875rem' },
            id: 'cell-color-encoding-select',
          }}
```
Might be a better way to do this but I like the look of this.  Feel free not to do this if you don't though
<img width="345" alt="Screen Shot 2021-01-21 at 10 06 23 PM" src="https://user-images.githubusercontent.com/43999641/105440679-001f3780-5c35-11eb-97c3-788e7c021393.png">